.rubocop.yml

```diff
@@ -46,3 +46,8 @@ Metrics/ParameterLists:
 
 Style/Lambda:
   Enabled: false
+
+# enable both hash-rocket and ruby19 hashmap syntax
+Style/HashSyntax:
+  Enabled: false
```

Comment:
Could you enable the cop, but set EnforcedStyle to `no_mixed_keys`? This supports both hash rockets and the 1.9+ style, but at least prevents both occurring within the same hash (https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/hash_syntax.rb#L44).